src/App.js

```diff
@@ -1,85 +1,273 @@
 import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { TodoList } from './components/TodoList/TodoList';
 
-function App() {
-  return (
-    <section className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
+class App extends React.Component {
+  state = {
+    task: '',
+    todos: JSON.parse(localStorage.getItem('state')) || [],
```

Comment:
all actions with localStorage we do in lifecycle methods

Reply:
fix